acceptance/lib/helper.rb

```diff
@@ -61,7 +61,9 @@ def self.get_option_value(value, legal_values, description,
 
     # precedence is environment variable, option file, default value
     value = ((env_var_name && ENV[env_var_name]) || value || default_value)
-    if value
+    if value == ""
+      value = default_value
```

Comment:
It seems like `to_sym` is already intelligently applied in this function.  We won't get `default_value.to_sym` when `default_value` is nil because it won't enter either the `if` or `elsif`, it'll fall through to the nil check below.